### PR TITLE
Experiences Listing for user Dashboard

### PR DESCRIPTION
**Pull request by patrickocoffeyo**
**This PR does the following:**
- Adds an API call, action, and reducer for experiences. The `experiences` state holds experiences owned by the current user, but does not include any referenced objects.

### To Review:

- [x] Checkout this branch.
- [x] Run `yarn start`.
- [x] Open your local dev instance.
- [x] Log into EditVR with the `mirzu` account (details in lastpass).
- [x] Note that you see two experiences in nice cards:
screen shot 2018-06-01 at 10 51 40

### CODE REVIEW

File: src/lib/api/axiosInstance.js [hunk 1]
```diff
@@ -66,12 +67,15 @@ const axiosInstance = (
...
       let r = response;
 
-      // TODO: This method will soon contain logic for denormalizing JSON-API
-      // responses, when the content type is correct.
       if (r.data) {
         r = r.data;
       }
 
+      if (format === 'jsonapi') {
+        const formatter = new Jsona();
```
Comment (techninja) [inline src/lib/api/axiosInstance.js]: Good find with Jsona!

File: src/lib/api/experiences.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+import { clientId } from '../../config';
+import axiosInstance from './axiosInstance';
+import { API_ENDPOINT_EXPERIENCES } from '../../constants';
+
+/**
+ * Fetches a list of experiences that are owned by the current user.
+ *
+ * @param {object} user - Object containing information about the current user.
```
Comment (techninja) [inline src/lib/api/experiences.js]: Docs don't match args here 😮

Comment (patrickocoffeyo) [reply src/lib/api/experiences.js]: Good catch! I'll fix shortly!

Comment (techninja) [reply src/lib/api/experiences.js]: Apparently not wrong! My bad.

**Pull request merged**